supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiAttributesSpec.scala

```diff
@@ -63,6 +63,14 @@ class PipelinesApiAttributesSpec extends FlatSpec with Matchers {
 
   }
 
+  it should "parse virtual-private-cloud" taggedAs IntegrationTest in {
```

Comment:
It looks like all the tests in this file are tagged as `IntegrationTest` which means they aren't going to be run regularly in any of our existing CI, though if they're just parsing config I'm not sure why they would be considered integration? 🤔